src/main/java/org/folio/circulation/LoanItemService.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.circulation;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ResultBinding.mapResult;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.LoanRepository;
+import org.folio.circulation.support.ItemRepository;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.http.client.Response;
+
+import java.util.concurrent.CompletableFuture;
+
+public class LoanItemService {
```

Comment:
Could this be renamed to be specific about its responsibility, e.g. StoreLoanAndItem?

Reply:
Renamed

Reply:
Renamed